ffcx/ir/representationutils.py

```diff
@@ -47,18 +46,18 @@ def id(self):
 def create_quadrature_points_and_weights(integral_type, cell, degree, rule):
     """Create quadrature rule and return points and weights."""
 
+    # from IPython import embed; embed()
     if integral_type == "cell":
-        (points, weights) = create_quadrature(cell.cellname(), degree, rule)
+        return create_quadrature(cell.cellname(), degree + 1, rule)
     elif integral_type in ufl.measure.facet_integral_types:
-        (points, weights) = create_quadrature(ufl.cell.cellname2facetname[cell.cellname()], degree,
-                                              rule)
+        return create_quadrature(ufl.cell.cellname2facetname[cell.cellname()], degree + 1, rule)
     elif integral_type in ufl.measure.point_integral_types:
-        (points, weights) = create_quadrature("vertex", degree, rule)
+        return create_quadrature("vertex", degree + 1, rule)
     elif integral_type == "expression":
-        (points, weights) = (None, None)
-    else:
-        logging.exception("Unknown integral type: {}".format(integral_type))
-    return (points, weights)
+        return (None, None)
+
+    logging.exception(f"Unknown integral type: {integral_type}")
```

Comment:
Is this ever hit?

Reply:
no. I've changed it to a RuntimeError as in the function below to give a stronger warning that something is wrong if this is hit